### ISSUE TITLE
Enable zstd compression with SBCL 2.2.6?

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#2365]
SBCL will soon be able to compress executables using Zstd (instead of zip)!
This means better compression ration and faster compression/decompression speed.

Zip compression used to significantly slow down Nyxt startup in the past, so we disabled it by default.
But maybe we can reenable it now if
```lisp
#+sbcl   
(when (uiop:version<= "2.2.6" (lisp-implementation-version))
  ...)
```
We would need to measure impact on the startup time.

The executable gets compression from ~130 MiB down to ~25 MiB.

**Comment by Ambrevar** [comment]
See https://bugs.launchpad.net/bugs/1881089 for the upstream issue.

**Comment by Ambrevar** [comment]
Just ran a quick test with SBCL 2.2.6.
To measure the startup time I ran 

```sh
time nyxt -I  -C --eval "17" --quit
```

No compression:
- Size: 140 MiB
- Time: ~ 200 ms.

Zstd compression level 3:
- Size 27 MiB
- Time: ~ 400 ms.

I tried different Zstd compression levels, but the default seems ideal.

So it effectively adds some 200 ms on my machine, possible a bit more on an slower system.
Seems still a bit much in my opinion, especially for script, so I think we should not enable it.

Thoughts?
